scripts/build.sh

```diff
@@ -140,6 +141,15 @@ function run_check_style() {
     cargo fmt -- --check
 }
 
+function run_lit_test() {
+    # We need a binary to execute the lit tests
+    cargo build
+    cargo build --release
```

Comment:
don't we need to the std lib here as well? which also needs to be packaged.. I know this is WIP just adding notes :D

Reply:
Also done. While we're at this, how important is a `ARCH` parameter here? I currently only pass the library and compiler locations to lit, which works because the hosts arch is assumed(?) see https://github.com/PLC-lang/rusty/blob/780a43bb16f03fc59882876daf4e49979af19ea8/scripts/build.sh#L150-L154

Reply:
if you don't pass arch during your build the host arch is assumed so not important. If you do pass the arch then the lib is saved in a sub folder of the output directory and needs to be passed

Reply:
So not important then for our CI (Github, Jenkins)?

Reply:
no, unless we add cross compile tests.. we can tackle that when we get there.